samples/crypto/aes_cbc/sample.yaml

```diff
@@ -9,6 +9,7 @@ tests:
     platform_allow: >
       nrf5340dk_nrf5340_cpuapp_ns nrf5340dk_nrf5340_cpuapp nrf9160dk_nrf9160_ns
       nrf9160dk_nrf9160 nrf52840dk_nrf52840 nrf9161dk_nrf9161 nrf9161dk_nrf9161_ns
+      nrf54h20pdk_nrf54h20_cpuapp nrf54l15pdk_nrf54l15_cpuapp
```

Comment:
When will we add nrf54l15pdk_nrf54l15_cpuapp_ns target for crypto samples?

Reply:
After https://nordicsemi.atlassian.net/browse/NCSDK-26277

Reply:
maybe a week or two